frontend/state/stores/runStore.js

```diff
@@ -28,17 +35,17 @@ class RunStore {
         this.runs[run.id] = new Run(run, socket);
     }
 
-    @action
+    @computed
     find(id) {
         return this.runs[id];
     }
 
-    @action 
+    @computed 
     getSortedRuns() {
```

Comment:
Should either be a getter:
```suggestion
    get sortedRuns() {
```

Or not a `computed` at all.

Reply:
Fixed this!